tests/integration/actions/settings/base.py

```diff
@@ -0,0 +1,133 @@
+"""Base class for ``settings`` interactive/stdout tests."""
```

Comment:
This is a module docstring, not a class docstring.
```suggestion
"""Base configuration for ``settings`` interactive/stdout tests."""
```

Reply:
done in next commit